Bio/GenBank/Scanner.py

```diff
@@ -522,8 +523,8 @@ def parse_cds_features(self, handle,
                     # SeqRecord objects cannot be created with annotations, they
                     # must be added afterwards.  So create an empty record and
                     # then populate it:
-                    record = SeqRecord(seq=None)
-                    annotations = record.annotations
+                    new_record = SeqRecord(seq=None)
```

Comment:
Yes, ``new_record`` is more explicit, but as ``record`` was the only SeqRecord variable in this method wasn't the shorter variable name clear enough?